torchci/scripts/td_rockset_analysis.py

```diff
@@ -0,0 +1,228 @@
+import copy
+import json
+import os
+from collections import defaultdict
+from functools import cmp_to_key
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from rockset_utils import query_rockset
+
+REPO_ROOT = Path(__file__).resolve().parent.parent.parent
+OUTPUT_FOLDER = REPO_ROOT / "logs" / "td_analysis"
```

Comment:
nit: consider the rename

```suggestion
OUTPUT_FOLDER = REPO_ROOT / "_logs" / "td_analysis"
```